autoload/db_ui/drawer.vim

```diff
@@ -719,6 +703,51 @@ function! s:drawer.get_buffer_name(db, buffer)
   return substitute(name, '^'.db_ui#utils#slug(a:db.name).'-', '', '')
 endfunction
 
```

Comment:
All the code in the new render methods below is copied verbatum from the original if/elseif/endif above.